src/natcap/invest/urban_nature_access.py

```diff
@@ -547,6 +655,376 @@ def execute(args):
     LOGGER.info('Finished Urban Nature Access Model')
 
 
+def _find_disjoint_polygon_groups(aoi_vector_path):
+    """Identify truly disjoint polygon groups.
+
+    Pygeoprocessing's calculate_disjoint_polygon_set() uses any form of
+    intersection (even if polygons touch but do not overlap) in determining
+    whether polygons are disjoint.  When a user provides a vector of AOIs that
+    touch but do NOT overlap, we do not need to use pygeoprocessing's set of
+    disjoint polygons, which makes more sense visually.
+
+    Args:
+        aoi_vector_path (string): A path to the AOI vector on disk.
+
+    Returns:
+        A list of lists of disjoint FIDs.
+    """
+    all_geometries = []
+    vector = ogr.Open(aoi_vector_path)
+    layer = vector.GetLayer()
+    area_sum = 0
+    all_fids = []
+    for feature in layer:
+        geom = feature.GetGeometryRef()
+        all_geometries.append(shapely.wkt.loads(geom.ExportToWkt()))
+        area_sum += geom.Area()
+        all_fids.append(feature.GetFID())
```

Comment:
I think in `_reproject_and_identify` we added our own ID field. Should we be relying on that here instead of FID?

Reply:
Ah this was within the disjoint polygon function, I think.  Probably, yes (might as well!), but I've removed the disjoint polygon handling per science team request.